lms/djangoapps/courseware/toggles.py

```diff
@@ -61,3 +61,41 @@ def course_exit_page_is_active(course_key):
         REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled(course_key) and
         COURSEWARE_MICROFRONTEND_COURSE_EXIT_PAGE.is_enabled(course_key)
     )
+
+# .. toggle_name: courseware.mfe_progress_milestones
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to display learner progress milestones in a course. Supports staged
+#    rollout to students for a new micro-frontend-based implementation of the courseware page.
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2020-10-01
+# .. toggle_target_removal_date: none
+# .. toggle_warnings: Also set settings.LEARNING_MICROFRONTEND_URL and
+#   ENABLE_COURSEWARE_MICROFRONTEND.
+# .. toggle_tickets: AA-371
+COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'mfe_progress_milestones',
+                                                                __name__)
+
+# .. toggle_name: courseware.mfe_progress_milestones_first_celebration
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to display a celebration modal on learner completion of their first section.
+#    Supports staged rollout to students for a new micro-frontend-based implementation of the courseware page.
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2020-10-01
+# .. toggle_target_removal_date: none
+# .. toggle_warnings: Also set settings.LEARNING_MICROFRONTEND_URL and
+#   ENABLE_COURSEWARE_MICROFRONTEND and COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES.
+# .. toggle_tickets: AA-371
+COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES_FIRST_SECTION_CELEBRATION = CourseWaffleFlag(
+    WAFFLE_FLAG_NAMESPACE,
+    'mfe_progress_milestones_first_celebration',
```

Comment:
I kind of think the "section" part is actually really important to this flag name. First celebration is pretty vague.

Reply:
Ooh yeah I added section but forgot it here